Ntuplizer/test/ntuplize_cfg.py

```diff
@@ -65,6 +65,11 @@
                  VarParsing.VarParsing.multiplicity.singleton,
                  VarParsing.VarParsing.varType.int,
                  "1 if gen-level ntuples are desired.")
+options.register('genLeptonType', 'hardProcessFS',
```

Comment:
This default causes a crash! I think you meant `fromHardProcessFS` (or to change the list of options below).